openupgradelib/openupgrade.py

```diff
@@ -657,12 +657,14 @@ def rename_fields(env, field_spec, no_deep=False):
         # TODO: Rename when the field is part of a submodel (ex. m2one.field)
         cr.execute("""
             UPDATE ir_filters
-            SET domain = replace(domain, %(old_pattern)s, %(new_pattern)s)
+            SET domain = regexp_replace(
+                domain, %(old_pattern)s, %(new_pattern)s
```

Comment:
```suggestion
                domain, %(old_pattern)s, %(new_pattern)s , 'g'
```
@MiquelRForgeFlow  May be need this cc @royleviindoo

Reply:
@duong77476 thank you, I added